src/overview/components/shareButtons.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react'
+import styles from './ShareButtons.css'
+
+class ShareButtons extends Component {
+    render() {
```

Comment:
If this is just a plain presentational view (no needed lifecycle methods or internal state), better to simplify it as a functional component. eg: 
```js
const ShareButtons = props => (
   <div>....</div>
)
```